sdk/python/feast/registry.py

```diff
@@ -577,10 +578,13 @@ def _write_registry_blob(self, registry_proto: RegistryProto):
 
     def _write_registry_json(self, registry_proto: RegistryProto):
         gs_bucket = self.gcs_client.get_bucket(self._bucket)
-        json_path = urlparse(f"{self._uri}/registry.json").path.lstrip("/")
+        json_path = urlparse(f"{self._uri_prefix}/registry_export.json").path.lstrip(
+            "/"
+        )
         blob = gs_bucket.blob(json_path)
         file_obj = TemporaryFile()
-        file_obj.write(bytes(MessageToJson(registry_proto), "utf-8"))
+        registry_dict = MessageToDict(registry_proto)
+        file_obj.write(bytes(json.dumps(registry_dict), "utf-8"))
```

Comment:
This results in a [JSONL](https://jsonlines.org/examples/) format. Change was made due to [the finding](https://stackoverflow.com/questions/51390797/json-formatting-error-when-loading-into-google-big-query) that pretty JSON is not supported by BQ. Note that a local `_write_registry_json` would still output the normal JSON format with mult-line objects.